### ISSUE TITLE
Github Actions workflow warnings

### ISSUE THREAD

**Issue by DevJackSmith** [m-h-c-t/mh-helper-extension#236]
```
[CD (16.x)](https://github.com/m-h-c-t/mh-helper-extension/actions/runs/3358478715/jobs/5565500376)
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, actions/setup-node, actions/upload-artifact, mikepenz/release-changelog-builder-action, actions/setup-node, actions/checkout
```
```
[CD (16.x)](https://github.com/m-h-c-t/mh-helper-extension/actions/runs/3358478715/jobs/5565500376#step:10:11)
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by hymccord** [comment]
For convenience. 
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

For even more convenience: 
A workflow using save-state or set-output like the following
```yml
- name: Save state
  run: echo "::save-state name={name}::{value}"

- name: Set output
  run: echo "::set-output name={name}::{value}"
```
should be updated to write to the new GITHUB_STATE and GITHUB_OUTPUT environment files:

```yml
- name: Save state
  run: echo "{name}={value}" >> $GITHUB_STATE

- name: Set output
  run: echo "{name}={value}" >> $GITHUB_OUTPUT
```

**Comment by DevJackSmith** [comment]
this needs to be checked/fixed across all github actions, not just publish.

**Comment by hymccord** [comment]
Closed via #367
